front/pages/api/w/[wId]/assistant/agent_configurations/index.ts

```diff
@@ -321,11 +321,22 @@ export async function createOrUpgradeAgentConfiguration({
       (action) => !action.description
     );
     if (actionsWithoutDescription.length) {
-      throw new Error(
-        `Every action must have a description. Missing descriptions for: ${actionsWithoutDescription
-          .map((action) => action.type)
-          .join(", ")}`
-      );
+      // We tolerate a missing description when editing am assistant with a single action of
```

Comment:
I think this should be removed because old single actions agents can become multi actions if I the user edits just to add a second action -> they will not understand that they have to open the first action to edit its description 😕

Reply:
Yep agreed